cpp/src/io/parquet/writer_impl.cu

```diff
@@ -2309,16 +2351,22 @@ writer::impl::impl(std::vector<std::unique_ptr<data_sink>> sinks,
     _dict_policy(options.get_dictionary_policy()),
     _max_dictionary_size(options.get_max_dictionary_size()),
     _max_page_fragment_size(options.get_max_page_fragment_size()),
-    _int96_timestamps(options.is_enabled_int96_timestamps()),
+    _int96_timestamps(options.is_enabled_int96_timestamps() and
```

Comment:
Is this okay or should we leave it as is and `CUDF_FAIL` when building arrow schema if `INT96` timestamps enabled?